modules/gdscript/gdscript.cpp

```diff
@@ -1994,8 +1994,9 @@ void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
 		"preload",
 		"signal",
 		"super",
-		"trait",
-		"yield",
+		"trait", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
+		"namespace", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
+		"yield", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
```

Comment:
idk

Reply:
Reserved for generators like in Python in Godot 4.x?